client/src/components/AddNewProfile.js

```diff
@@ -173,6 +227,18 @@ const AddNewProfile = () => {
           options={statusOptions}
         />
       </Form.Field>
+      <Form.Field>
+        <label>Nationality</label>
+        <Dropdown
+          name="nationality"
```

Comment:
```suggestion
        <label for="nationality">Nationality</label>
        <Dropdown
          id="nationality"
          name="nationality"
```
Every label should be connected to its input, and we can do this by adding `for` which points to the `id` of the input. This means people using screen readers also know what the label is, and clicking on the label also focuses on the input.